### ISSUE TITLE
Test Failure: com.ibm.ws.logging.fat.StackTraceFilteringForLoggedExceptionParametersTest.testConsoleIsTrimmedForLoggedParameter

### ISSUE THREAD

**Issue by fmhwong** [OpenLiberty/open-liberty#19506]
RTC 287983

```
testConsoleIsTrimmedForLoggedParameter:junit.framework.AssertionFailedError: 2021-11-26-06:00:13:879 The console stack was not trimmed. (could not find 'at \[internal classes\]')
at com.ibm.ws.logging.fat.AbstractStackTraceFilteringTest.assertConsoleLogContains(AbstractStackTraceFilteringTest.java:70)
at com.ibm.ws.logging.fat.StackTraceFilteringForLoggedExceptionParametersTest.testConsoleIsTrimmedForLoggedParameter(StackTraceFilteringForLoggedExceptionParametersTest.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:198)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:319)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:172)

```

**Comment by ellen-lau** [comment]
This issue started occurring at the same time as JVM updates:

IBM Semeru JDK 8 Open -> 8U312B07 OpenJ9 0.29.0
IBM Semeru JDK 11 Open -> 11.0.13+8 OpenJ9 0.29.0
IBM Semeru JDK 17 Open -> 17.0.1+12 OpenJ9 0.29.1 M1

Related logs show:
```
at <unknown class>.<init>(Cookie.java:175)
	at com.ibm.ws.logging.fat.broken.servlet.SpecUsingServlet.doGet(SpecUsingServlet.java:46)
	at <unknown class>.service(HttpServlet.java:687)
	at <unknown class>.service(HttpServlet.java:790)
	at <unknown class>.service(ServletWrapper.java:1258)
	at <unknown class>.handleRequest(ServletWrapper.java:746)
	at <unknown class>.handleRequest(ServletWrapper.java:443)
```

This issue was reported in the Open J9 repo under: https://github.com/eclipse-openj9/openj9/issues/13759, and should be fixed in Release 0.30 sometime in January.

**Comment by ellen-lau** [comment]
The issue has been resolved after the JDK update to Release 0.30.
